packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -22,22 +30,52 @@ export function collectDependencies({ onlyNative, outputDir, widgetName }) {
             const resolvedPackagePath = await resolvePackage(
                 source,
                 dirname(importer ? importer : rollupOptions.input[0])
-            );
-            if (resolvedPackagePath && (!onlyNative || (await hasNativeCode(resolvedPackagePath)))) {
-                if (!managedDependencies.includes(resolvedPackagePath)) {
-                    managedDependencies.push(resolvedPackagePath);
+            ).catch(e => {
+                console.warn(e.message);
+                return null;
+            });
+
+            if (resolvedPackagePath) {
+                const checkOnlyNativeAndHasNativeCode = !onlyNative || (await hasNativeCode(resolvedPackagePath));
```

Comment:
This var name is confusing to me. It's called "check only native *and* has native code" where it is describing "doesnt have only native code *or* some native code"

Reply:
I've changed it. Could you check it again, please?